.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,244 @@
+
+      - name: Run Unit Tests
+        run: |
+          export SBT_OPTS="$SBT_OPTS"
+          ./.github/run-unit-test-selection
+
+
+  build_docker_image:
+    name: Build Docker Image
+    needs: build_base
+    runs-on: ubuntu-latest
+    container: rchain/buildenv
+    steps:
+      - name: Load Working Tree
+        uses: actions/download-artifact@v1
+        with:
+          name: rchain-worktree
+
+      - name: Restore Working Tree
+        run: tar -H posix -xzf rchain-worktree/rchain-worktree.tar.gz
+
+      - name: Initialize Environment
+        run: ./.github/init-environment
+
+      - name: Restore Cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.CACHE_ROOT }}
+          key: scala-${{ env.CACHE_TAG_SCALA }}
+          restore-keys: scala-
+
+      - name: Create Docker Image
+        run: sbt node/docker:publishLocal
+
+      - name: Export Docker Image
+        run: docker image save coop.rchain/rnode | gzip >../rnode-docker.tar.gz
+
+      - name: Save Docker Image
+        uses: actions/upload-artifact@v1
+        with:
+          name: artifacts-docker
+          path: ../rnode-docker.tar.gz
+
+
+  build_packages:
+    name: Build Packages
+    needs: build_base
+    runs-on: ubuntu-latest
+    container: rchain/buildenv
+    steps:
+      - name: Load Working Tree
+        uses: actions/download-artifact@v1
+        with:
+          name: rchain-worktree
+
+      - name: Restore Working Tree
+        run: tar -H posix -xzf rchain-worktree/rchain-worktree.tar.gz
+
+      - name: Initialize Environment
+        run: ./.github/init-environment
+
+      - name: Restore Cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.CACHE_ROOT }}
+          key: scala-${{ env.CACHE_TAG_SCALA }}
+          restore-keys: scala-
+
+      - name: Create Packages
+        run: |
+          sbt node/universal:packageZipTarball \
+              node/debian:packageBin \
+              node/rpm:packageBin
+          mkdir ../packages
+          cp node/target/universal/rnode-*.tgz \
+              node/target/rnode_*.deb \
+              node/target/rpm/RPMS/noarch/rnode-*.rpm \
+              ../packages/
+
+      - name: Save Packages
+        uses: actions/upload-artifact@v1
+        with:
+          name: artifacts-packages
+          path: ../packages/
+
+
+  run_integration_tests:
+    name: Integration Tests
+    needs: build_docker_image
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tests:
+          - test_backward_compatible
+          - test_dag_correctness
+          - test_finalization
+          - test_genesis_ceremony
+          - test_propose
+          - test_slash_invalid_block_hash
+          - test_slash_invalid_block_number
+          - test_slash_invalid_block_seq
+          - test_slash_justification_not_correct
+          - test_slash_GHOST_disobeyed
+          - test_storage
+          - test_synchrony_constraint
+          - test_wallets
```

Comment:
So if I add a new test , I should put the test name here, right? 
Is there a more intelligent way  to do that?

Reply:
After reading your comments in discord, I totally get it~
If I don't specify the test here ,all other tests would run in `REMAINER`. AM I right?

Reply:
You're right.